packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,362 @@
+        <View style={styles.descriptionContainer}>
+          <Text style={{ ...fontStyles.h1, flex: 1 }}>{t('dappsScreen.message')}</Text>
+          <View style={styles.descriptionImage}>
+            <DappsExplorerLogo />
+          </View>
+        </View>
+        <>
+          {loading && (
+            <View style={styles.centerContainer}>
+              <ActivityIndicator
+                style={styles.loadingIcon}
+                size="large"
+                color={colors.greenBrand}
+                testID="DAppExplorerScreen/loading"
+              />
+            </View>
+          )}
+          {!loading && error && (
+            <View style={{ ...styles.centerContainer, marginTop: 96 }}>
+              <Text style={fontStyles.regular}>{t('dappsScreen.errorMessage')}</Text>
+            </View>
+          )}
+          {!loading &&
+            !error &&
+            result &&
+            result.map((category: CategoryWithDapps) => (
+              <CategoryWithDapps category={category} onDappPress={onDappPress} />
+            ))}
+        </>
+      </ScrollView>
+    </SafeAreaView>
+  )
+}
+
+function CategoryWithDapps({ category: categoryWithApps, onDappPress }: CategoryWithDappsProps) {
+  Logger.debug(TAG, `Render category ${JSON.stringify(categoryWithApps)}`)
+
+  return (
+    <View style={styles.categoryContainer} key={`category-${categoryWithApps.id}`}>
+      <View
+        style={{
+          ...styles.categoryTextContainer,
+          backgroundColor: categoryWithApps.backgroundColor,
+        }}
+      >
+        <Text style={{ ...styles.categoryText, color: categoryWithApps.fontColor }}>
+          {categoryWithApps.name}
+        </Text>
+      </View>
+      <>
+        {categoryWithApps.dapps.map((dapp) => (
+          <Dapp dapp={dapp} onDappPress={onDappPress} />
+        ))}
+      </>
+    </View>
+  )
+}
+
+function Dapp({ dapp, onDappPress }: DappProps) {
+  Logger.debug(TAG, `Render item ${JSON.stringify(dapp)}`)
+
+  const onPress = () => {
+    return onDappPress(dapp)
+  }
+
+  return (
+    <Card style={styles.card} rounded={true} shadow={Shadow.Soft} key={`item-${dapp.id}`}>
+      <TouchableOpacity style={styles.pressableCard} onPress={onPress}>
+        <Image source={{ uri: dapp.iconUrl }} style={styles.dappIcon} />
+        <View style={styles.itemTextContainer}>
+          <Text style={styles.itemTitleText}>{dapp.name}</Text>
+          <Text style={styles.itemSubtitleText}>{dapp.description}</Text>
+        </View>
+        <LinkArrow style={styles.linkArrow} />
+      </TouchableOpacity>
+    </Card>
+  )
+}
+
+const styles = StyleSheet.create({
+  safeAreaContainer: {
+    flex: 1,
+  },
+  scrollContainer: {
+    flex: 1,
+    flexDirection: 'column',
+  },
+  centerContainer: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    flex: 1,
+  },
+  categoryContainer: {
+    alignItems: 'flex-start',
+    justifyContent: 'center',
+    flex: 1,
+    marginVertical: 16,
```

Comment:
We haven't used this everywhere sofar, but we have standard spacing constants. For instance: https://github.com/valora-inc/wallet/blob/c1e0fae65108e064ce313f56ae85cbba84b1810f/packages/mobile/src/verify/VerificationLoadingScreen.tsx#L290